app/controllers/admin_users/sessions_controller.rb

```diff
@@ -0,0 +1,8 @@
+module AdminUsers
+  class SessionsController < Devise::SessionsController
+    def new
+      @show_sign_in = !Rails.env.production?
```

Comment:
There is an environment variable `allowAdminPassword` in `helm_deploy/apply-for-legal-aid/values-uat.yaml:102` that may be more appropriate for this check. It is set as `true` for UAT and `false` for staging and prod.